build.zig

```diff
@@ -723,6 +728,30 @@ pub fn ldcBuildStep(b: *Build, options: DCompileStep) !*Build.Step.InstallDir {
     const cpu_model = options.target.result.cpu.model.llvm_name orelse "generic";
     ldc_exec.addArg(b.fmt("-mcpu={s}", .{cpu_model}));
 
+    // zig workaround for ldc2 get cpu-features
+    var cpu_args = std.ArrayList(u8).init(b.allocator);
+    defer cpu_args.deinit();
+    const all_features_list = options.target.result.cpu.arch.allFeaturesList();
+    for (all_features_list, 0..) |feature, index_usize| {
+        const index = @as(std.Target.Cpu.Feature.Set.Index, @intCast(index_usize));
+        const is_enabled = options.target.result.cpu.features.isEnabled(index);
+        if (feature.llvm_name) |llvm_name| {
+            const plus_or_minus = "-+"[@intFromBool(is_enabled)];
+            if (is_enabled) {
+                if (std.mem.endsWith(u8, llvm_name, "contextidrel2")) {
```

Comment:
Failure to capture flag correctly.
Unfortunately libsokol contains system-dependencies making it hard to cross-compile outside macOS.